site/Makefile

```diff
@@ -2,15 +2,36 @@ ROOT_DIR := ..
 include $(ROOT_DIR)/Makefile.env
 include $(ROOT_DIR)/hack/make-rules/tools.mk
 
-
-CONNECTORS_PROTO := $(ROOT_DIR)/pkg/connectors/protos
+ABS_ROOT_DIR := $(abspath ${ROOT_DIR})
+OPENAPI_GENERATOR_IMG := openapitools/openapi-generator-cli:v5.3.1
 
 .PHONY: generate
-generate: $(TOOLBIN)/crdoc $(TOOLBIN)/protoc $(TOOLBIN)/protoc-gen-doc
+generate: generate-crd-docs generate-connectors-docs
+
+.PHONY: generate-crd-docs
+generate-crd-docs: $(TOOLBIN)/crdoc
 	PATH=$(TOOLBIN) crdoc --template ./templates/crd/main.tmpl --resources $(ROOT_DIR)/charts/fybrik-crd/templates --output ./docs/reference/crds.md
-	PATH=$(TOOLBIN) protoc -I=$(CONNECTORS_PROTO) --doc_out=./docs/reference/ --doc_opt=./templates/proto/protoc.tmpl,connectors.md $(sort $(wildcard ./$(CONNECTORS_PROTO)/*.proto))
-	go mod tidy
+
+.PHONY: generate-connectors-docs
+generate-connectors-docs:
+	rm -rf ./docs/reference/connectors-*
+
+	docker run --user $(shell id -u):$(shell id -g) --rm -v "${ABS_ROOT_DIR}":/workdir \
```

Comment:
For development I use a shared directory between my Windows machine and VBox. Therefore, the project directory owner is "root" and group is "vboxsf". Running docker command with `--user $(shell id -u):$(shell id -g)` returns `[error] The spec file is not found: /workdir/connectors/api/datacatalog.spec.yaml` due to wrong directories permissions. 
Therefore, I suggest to provide the following `--user` flag `--user $(shell id -u):$(shell  stat -c %g "${ABS_ROOT_DIR}")`

Reply:
fixed. Please check. Thanks.